src/blocks/HedvigersBlock/Mobile/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { fonts } from '@hedviginsurance/brand'
+// @ts-ignore
```

Comment:
What's up here? 🙂

Reply:
no types and i was lazy haha. Will fix

Reply:
actually, diving into this. Would it be _ok_ to skip? `swipeable-views` literally has 100 lines of proptypes, defining the props for this dep will be a hassle

Reply:
actually, diving into this. Would it be _ok_ to skip? `swipeable-views` literally has 100 lines of proptypes, defining the props for this dep will be a hassle

Reply:
https://www.youtube.com/watch?v=-lI3_b_bHLY

Reply:
(go ahead and keep the ignore 😉 )